runtime/onert/core/src/exec/ExecutorBase.h

```diff
@@ -158,6 +158,11 @@ class ExecutorBase : public IExecutor
   void handleDynamicInputTensor(ir::IOIndex input_index, const IODescription &desc);
 };
 
+/**
+ * @brief Returns @c true if any input tensor is dynamic; @c false if all are static tensors
+ */
+bool hasDynamicTensor(const std::vector<std::shared_ptr<onert::backend::ITensor>> &input_tensors);
```

Comment:
Where will you use this function?

Reply:
Could you find the code in https://github.com/Samsung/ONE/pull/2957/files#diff-d508434b4530bfdc88a7965944a65966R55 ?

Reply:
If so, how about moving this function in `ExecutorBase` as protected method?

Reply:
That would be better. Thanks. Fixed and committed.